### PR TITLE
ACS-1185 Update activemq image to 5.16.1

### PR DESCRIPTION
**Pull request by PunchedChimera**
See [ACS-1185](https://alfresco.atlassian.net/browse/ACS-1185)

### CODE REVIEW

**Review by TheOutsSeeker — APPROVED** [review]
Thanks for the update.

File: alfresco-transform-tika/alfresco-transform-tika-boot/pom.xml [hunk 1]
```diff
@@ -97,7 +97,7 @@
         <dependency>
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcmail-jdk15on</artifactId>
-            <version>1.68</version>
+            <version>1.64</version>
```
Comment (montgolfiere) [inline alfresco-transform-tika/alfresco-transform-tika-boot/pom.xml:100]: Downgrade ?

**Review by montgolfiere — CHANGES_REQUESTED** [review]
See comment @PunchedChimera ... does this branch need to be updated from master ?

File: alfresco-transform-tika/alfresco-transform-tika-boot/pom.xml (hunk 1 above)
Comment (PunchedChimera) [reply alfresco-transform-tika/alfresco-transform-tika-boot/pom.xml:100]: Hmm, not sure how that entered... I'll fix now

**Review by KikoUK — APPROVED** [review]

**Review by montgolfiere — APPROVED** [review]

**Pull request merged**